### ISSUE TITLE
Error using InjectManifest pwa mode

### ISSUE THREAD

**Issue by lukadriel7** [freddy38510/quasar-app-extension-ssg#31]
Hello, While building using the InjectManifest with pwa:true for ssr, the following error occurs:

`Extension(ssg) · ⚠️    ValidationError: "swSrc" is not a supported parameter.`

This is my custom manifest:

```
/*
 * This file (which will be your service worker)
 * is picked up by the build system ONLY if
 * quasar.conf > pwa > workboxPluginMode is set to "InjectManifest"
 */

import * as firebase from "firebase/app";
import "firebase/messaging";

import { setCacheNameDetails, skipWaiting, clientsClaim } from "workbox-core";
import { precacheAndRoute, createHandlerBoundToURL } from "workbox-precaching";
import { registerRoute, NavigationRoute } from "workbox-routing";
import { CacheFirst } from "workbox-strategies";

setCacheNameDetails({ prefix: "quasar-app" });
clientsClaim();
skipWaiting();

self.__precacheManifest = [].concat(self.__WB_MANIFEST || []);
precacheAndRoute(self.__precacheManifest);

registerRoute(
  ({ request }) => request.destination === "image",
  new CacheFirst()
);

registerRoute(
  new NavigationRoute(createHandlerBoundToURL("/offline.html"), {
    denylist: [/service-worker\.js$/, /workbox-(.)*\.js$/]
  })
);
// Initialize the Firebase app in the service worker by passing in
// your app's Firebase config object.
// https://firebase.google.com/docs/web/setup#config-object
firebase.initializeApp({
  
});

// Retrieve an instance of Firebase Messaging so that it can handle background
// messages.
const messaging = firebase.messaging();
```

The error doesn't occur when using GenerateSW

**Comment by freddy38510** [comment]
@lukadriel7,

Thanks for opening this issue. It should be fixed in the new [`1.0.2` release](https://www.npmjs.com/package/quasar-app-extension-ssg/v/1.0.2).

Could you try it by upgrading the app extension with this command please ?
```bash
quasar ext add ssg
```

Also, like it is mentioned in the README, `offline.html` file is replaced by your [configured fallback html file](https://github.com/freddy38510/quasar-app-extension-ssg#fallback) (default is `404.html`).

So, to make navigation fallback with PWA works, you should do that:

```javascript
registerRoute(
  new NavigationRoute(createHandlerBoundToURL("/404.html"), {
    denylist: [/service-worker\.js$/, /workbox-(.)*\.js$/]
  })
);
```

Maybe, i should make this information more visible in the README.

**Comment by lukadriel7** [comment]
Thank you very much for the quick fix and the notice.
Thank you also for your work on this package.
